src/python/grpcio/grpc/_channel.py

```diff
@@ -24,6 +24,7 @@
 from grpc._cython import cygrpc
 from grpc.framework.foundation import callable_util
 
+logging.basicConfig()
```

Comment:
It's a little weird to see these calls inside the library modules - I sort of get the sense from the function's documentation that it's of global effect and is only useful once per Python interpreter lifetime. If that's the case, what would you think of putting these calls in the test modules? Something like
```python
if __name__ == '__main__':
    logging.basicConfig()
    unittest.main(verbosity=2)
```
?

Incidentally, where do other Python codebases put their calls to this function?